src/test/ui/span/issue-29595.stderr

```diff
@@ -0,0 +1,10 @@
+error[E0277]: the trait bound `u8: Tr` is not satisfied
+  --> src\test\ui\span\issue-29595.rs:18:17
```

Comment:
[This should be `$DIR/issue-29595.rs:18:17`](https://travis-ci.org/rust-lang/rust/jobs/210693743#L4363).